tests/queries/0_stateless/02368_cancel_write_into_hdfs.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest, no-stress
```

Comment:
Why `no-stress`? This tag should not exists at all.

Reply:
Because we don't setup local hdfs cluster in stress tests. And queries from this test executes really long in case when there is no hdfs cluster, because they do a lot of retries and uses shared mutex between all calls somhere deep inside hdfs lib (in this test we have a lot of queries to hdfs) and this leads to failed stress test, because we think that these queries hung.
We just need to setup hdfs cluster in stress tests similar to stateless tests. Don't remember why I didn't do it, I huess I should've.

Reply:
As I remember, these retries occures somehere deep in RPC code in hdfs lib and it's not so easy. But I will check if I can fix them